src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminAuditService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.dts.fact.services.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.dts.fact.entity.Audit;
+import uk.gov.hmcts.dts.fact.repositories.AuditRepository;
+import uk.gov.hmcts.dts.fact.repositories.AuditTypeRepository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@SuppressWarnings("PMD.UseObjectForClearerAPI")
+public class AdminAuditService {
+
+    private final AuditRepository auditRepository;
+    private final AuditTypeRepository auditTypeRepository;
+
+    @Autowired
+    public AdminAuditService(AuditRepository auditRepository, AuditTypeRepository auditTypeRepository) {
+        this.auditRepository = auditRepository;
+        this.auditTypeRepository = auditTypeRepository;
+    }
+
+    public List<uk.gov.hmcts.dts.fact.model.admin.Audit> getAllAuditData() {
```

Comment:
As per comment in controller, returning the data in paginated form might be more useful when it comes to doing the front-end for this.